apps/carbon/index.html

```diff
@@ -1,13 +1,16 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="src/assets/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>React 18, TypeScript, Vitest, Testing Library, TailwindCSS 3, Vite, Eslint, Juno UI components and Prettier boilerplate</title>
-  </head>
-  <body>
-    <div id="root"></div>
-    <script type="module" src="/src/index.tsx"></script>
-  </body>
-</html>
+
+<head>
+	<meta charset="UTF-8" />
+	<link rel="icon" type="image/svg+xml" href="src/assets/favicon.ico" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<title>Carbon</title>
+</head>
+
+<body>
+	<div id="root"></div>
+	<script type="module" src="/src/index.tsx"></script>
+</body>
+
+</html>
```

Comment:
missing new line. It looks like you’re using different formatting rules—an indent of 4 instead of 2.

Reply:
yes vscode formatting removed it and it did it only for html file. I'll check settings.

Reply:
> yes vscode formatting removed it and it did it only for html file. I'll check settings.

I've experienced the same issue. The HTML files are recognized as Nunjucks, which causes the last line to be removed. Changing the file format to HTML resolves it for me.

Reply:
I discovered why new lines in HTML files are removed when formatted. Even though the `.prettierrc` file includes the rule `"endOfLine": "lf"` VS Code overrides this rule and formats HTML files using its default settings. If you’re facing this issue, here are a few possible solutions:

**- Apply Prettier Rules at the End:**
(I couldn’t get this to work)
**- Disable VS Code’s HTML Formatter in Workspace Settings:**
(I didn’t prefer this solution)
<img width="572" alt="Screenshot 2024-10-25 at 15 00 27" src="https://github.com/user-attachments/assets/d7a869a3-d2ce-41ac-883b-e6eedf6ab63c">

**- Override the Default HTML Formatter with Prettier in Workspace Settings JSON:**
```"[html]": { "editor.defaultFormatter": "esbenp.prettier-vscode"},```
(This is the solution I chose)